build.cake

```diff
@@ -311,6 +311,24 @@ Task("Publish-Chocolatey")
     }
 });
 
+Task("Create-Release-Notes")
+  .Does(() =>
+{
+    var userName = EnvironmentVariable("CAKE_GITHUB_USERNAME");
+    var password = EnvironmentVariable("CAKE_GITHUB_PASSWORD");
+    var owner = EnvironmentVariable("CAKE_GITHUB_OWNER");
+    var repo = EnvironmentVariable("CAKE_GITHUB_REPO");
+    var targetBranch = EnvironmentVariable("CAKE_TARGET_BRANCH");
```

Comment:
What happens if this isn't set for some reason? Shouldn't this always be `main`?